css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,14 @@
+button:hover {
```

Comment:
Let's give our button some paddings to make it more like our example when it's not hovered :)

Reply:
@BenGrimmius maybe we missed this one? :)